### DISCUSSION TITLE
Usage of pre-installed MultiDiffusion ?

### DISCUSSION THREAD

**Discussion by devils-shadow** [lllyasviel/stable-diffusion-webui-forge#403] [Q&A]
What's the best-case usage scenario of the multidiffusion addon? I've been trying to use it via txt2img but all it does it stitch the same image multiple times. Is there any way to prompt every individual region? Or am I misunderstanding its usage case?

**Comment by catboxanon** [comment]
The regional prompting feature of MultiDiffusion is not currently implemented in the built-in extension. https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/191
